cosmwasm/ibc-union/app/ucs06-funded-dispatch/src/types.rs

```diff
@@ -0,0 +1,50 @@
+use alloy::primitives::U256;
+use cosmwasm_std::{Addr, Uint128};
+
+pub const FLAG_ALLOW_FAILURE: U256 = U256::from_be_slice(&[1]);
+pub const FLAG_ALLOW_MARKET_MAKER: U256 = U256::from_be_slice(&[2]);
+
+alloy::sol! {
+    #[derive(Debug)]
+    struct FundedDispatchFund {
+        bytes token;
+        uint256 amount;
+    }
+
+    #[derive(Debug)]
+    struct FundedDispatchParameters {
+        uint256 flags;
+        FundedDispatchFund[] funds;
+        bytes contractAddress;
```

Comment:
NIT: Make these fields snake case to be consistent with ZKGM and Rust conventions